extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -100,51 +102,73 @@ static public function canShowDiscussions( $cityId, $ignoreWgEnableRecirculation
 	}
 
 	private static function addLiftIgniterMetadata( OutputPage $outputPage ) {
-		global $wgCityId,
-		       $wgDevelEnvironment,
-		       $wgEnableArticleFeaturedVideo,
-		       $wgIsPrivateWiki,
-		       $wgLanguageCode,
-		       $wgStagingEnvironment,
-		       $wgWikiaEnvironment;
+		$metaData = self::getMetaData();
+		$metaDataJson = json_encode( $metaData );
```

Comment:
imo at least one of this variables might be inlined, so consder e.g. `metadataJson = json( self:: metaData() )`